fff

```diff
@@ -154,9 +165,30 @@ get_mime_type() {
     mime_type=$(file "-${file_flags:-biL}" "$1" 2>/dev/null)
 }
 
+create_marked_files_cache () {
+    touch "$marked_files_cache" || die "Could not touch $marked_files_cache ."
+    test -f "$marked_files_cache" || die "$marked_files_cache is not a file."
+    chmod 600 "$marked_files_cache" || die "Could not set chmod 0600 on $marked_files_cache ."
+    chown "$USER" "$marked_files_cache" || die "Could not set owner of $marked_files_cache ."
```

Comment:
How does the terminal actually look after such error conditions? fff resets it on exit which can interfere with messages.